src/main/java/step5/domain/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package step5.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingGame {
+
+    List<RacingCar> carList = new ArrayList<>();
+
+    public RacingGame() {}
+
+    public List<RacingCar> getCarList() {
+        return carList;
+    }
+
+    public void addRacingCar(RacingCar car) {
+        carList.add(car);
+    }
+
+    public int getMaxPosition() {
+        return carList
+                .stream()
+                .max(Comparator.comparing(RacingCar::getPosition))
+                .get().getPosition();
```

Comment:
전체적으로 스트림 사용도 잘 해주셨네요. 👍 👍  
```suggestion
        return carList.stream()
                .mapToInt(RacingCar::getPosition)
                .max().orElse(0);
```
해당 로직은 이런식으로 표현 가능해요. 참고만해주세요.